src/components/sbb-menu/sbb-menu.tsx

```diff
@@ -336,6 +356,42 @@ export class SbbMenu implements ComponentInterface {
     this._element.style.setProperty('--sbb-menu-max-height', menuPosition.maxHeight);
   }
 
+  /**
+   * Create an array with only the sbb-menu-action children
+   */
+  private _readActions(): void {
```

Comment:
This is still a draft PR, so my comment is not intended as a final verdict.
I would probably prefer to split detection and rendering, e.g. in `_readActions` you assign `this._content` with an array of `HTMLSbbMenuAction` (and maybe rename to `this._actions`?) and undefined/null in the case when not every child is a `<sbb-menu-action>` instance.

Then in `render` you render depending on whether `this._actions` is defined or not.

What do you think?

Reply:
Done and done, looks better already